CDCgov/NEDSS-Modernization
# Route libraries to classic

## Description

The library links in the modernized Page Builder UI should route to the Classic™ pages.

While updating the `Navigation` component, it was pointed out to me that both Patient Search and Page Builder have implemented `Side Navigation` components. This PR creates a singular `SideNavigation` component that will be shared.
[USWDS Side Navigation](https://designsystem.digital.gov/components/side-navigation/)
[Figma](https://www.figma.com/file/Uz9YzHbHQAubHpQTcsibA2/NBS-PageBuilder-2.0?type=design&node-id=4927-116683&mode=dev)

## Tickets

* [CNFT2-1417](https://cdc-nbs.atlassian.net/browse/CNFT2-1417)

## Steps to verify

1. Navigate to http://localhost:8080/nbs/login
2. Login as `superuser`
3. Navigate to System Management
4. Click `Page Management` -> `Manage Pages`
5. Click `Templates`, `Conditions`, `Questions`, `Value sets`
6. Verify the appropriate Classic™ library is loaded for each



![routelibraries](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/51973a94-2326-408f-932e-b0246bf3bfa5)


[CNFT2-1417]: https://cdc-nbs.atlassian.net/browse/CNFT2-1417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT2-1417]: https://cdc-nbs.atlassian.net/browse/CNFT2-1417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
